src/BowlingGame.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+var BowlingGame = function () {
+  this.LOWESTSCORE = 0;
+  this.HIGHESTSCORE = 270;
+  this.MAXFRAMES = 10;
+  this.score = 0;
+  this.noOfFrames = [];
+}
+
+BowlingGame.prototype.addFrame = function (frame) {
+  if (this._addFrameLimit() === true) {
```

Comment:
Could `_addFrameLimit` have a clearer name that reflects what it means in terms of the status of the game?